DisguiseUnityRenderStream/Runtime/PipelineAbstraction/CameraCapturePresenter.cs

```diff
@@ -41,17 +41,37 @@ public enum Mode
         
         CameraCapture m_cameraCapture;
         
+        public CameraCapture cameraCapture
+        {
+            get => m_cameraCapture;
+            set
+            {
+                m_cameraCapture = value;
+                Assign(m_cameraCapture);
+            }
+        }
+        
         protected override void OnEnable()
         {
             base.OnEnable();
 
-            m_cameraCapture = GetComponent<CameraCapture>();
+            if (m_cameraCapture == null)
+                m_cameraCapture = GetComponent<CameraCapture>();
```

Comment:
I don't think I understand this part. We're setting the `m_captureCapture` field here but we're also setting it from `DisguisePresenter`. What is the intent?

Reply:
It's a middleground: we can either auto-assign the sibling CameraCapture (old behavior) or set one from a different GameObject (like in the DisguisePresenter prefab).

It's for us devs only. When I test and prototype changes for the pipeline abstraction I use a basic setup: CameraCapture + CameraCapturePresenter on the same GameObject.